src/main/scala/it/pagopa/interop/catalogprocess/api/impl/ProcessApiServiceImpl.scala

```diff
@@ -67,6 +67,20 @@ final case class ProcessApiServiceImpl(
     val result: Future[EService] = for {
       organizationId <- getOrganizationIdFutureUUID(contexts)
       clientSeed = Converter.convertToClientEServiceSeed(eServiceSeed, organizationId)
+      maybeEservice <- ReadModelQueries.listEServices(
+        eServiceSeed.name.some,
+        Seq.empty,
+        Seq(clientSeed.producerId.toString),
+        Seq.empty,
+        0,
+        1,
+        true
```

Comment:
If we're picking this way, i.e. using a boolean parameter, let's at least name it
```suggestion
        exactMatchOnName = true
```